submitter/jobsubmitter.py

```diff
@@ -104,6 +104,16 @@ def _prepare_directories(self):
             os.mkdir(self.job_tmp_dir)
 
     def _command_line(self):
+        """
+        Start ACCESS-specific code
+        In addition to different arguments and required bins, ACCESS requires a specific version of Toil that prevents the following PATH from recursively appending itself on every job, causing "OSExit Too many argument".
+        """
+        path = "PATH=/work/access/testing/users/fraihaa/ridgeback/conda/envs/toil-msk-3.21.1-MSK-rc1/bin:/conda/bin:/home/accessbot/miniconda3/envs/ACCESS_1.3.26/bin/:{}".format(os.environ.get('PATH'))
+        command_line = [path, 'toil-cwl-runner', '--logFile', 'toil_log.log', '--batchSystem','lsf','--disable-user-provenance','--logLevel', 'DEBUG','--disable-host-provenance','--stats', '--debug', '--cleanWorkDir', 'always', '--disableCaching', '--preserve-environment', 'PATH', 'TMPDIR', 'TOIL_LSF_ARGS', 'SINGULARITY_PULLDIR', 'SINGULARITY_CACHEDIR', 'PWD', '_JAVA_OPTIONS', 'PYTHONPATH', 'TEMP', '--defaultMemory', '10G', '--realTimeLogging', '--jobStore', self.job_store_dir, '--tmpdir-prefix', self.job_tmp_dir, '--workDir', self.job_work_dir, '--outdir', self.job_outputs_dir]
+        """
+        End ACCESS-specific code
+        """
```

Comment:
There's no condition, wanted to get feedback as to the preferred approach -- whether I can just check the string of the inputs or filepaths rather than pass a flag in the request param. I'm also curious if there's a preference on where to put the bin folders we're passing to PATH.

Reply:
I think the cleanest way to implement the condition would be to use the github.repository value using self.app to check for the ACCESS pipeline. This will link the pipeline github repository with that function in ridgeback. I think checking the string of the inputs might cause unintentional collisions.